flytekit/annotated/launch_plan.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import annotations
```

Comment:
why __future__

Reply:
this is what you do if you want delayed type checking in 3.7, so that you can do `-> ClassName` instead of `-> "ClassName"`

Reply:
aah TIL